cockpit/gui/shellWindow.py

```diff
@@ -77,6 +77,6 @@ def OnDropFiles(self, x, y, filenames):
 
 
 def makeWindow(parent):
-    shell = ShellWindow(parent, title = "Python shell",
+    shell = ShellWindow(None, title = "Python shell",
```

Comment:
Should we then just remove the `parent` argument from this function? It was only used here.

Reply:
Maybe, but that would make it different from the makeWindow definitions in other UI modules. It's probably better to revert this change and make a change to pass None where makeWindow is called.

Reply:
New issue opened #474

Reply:
New issue #474